### PR TITLE
GoshkoHW3

### PR DESCRIPTION
**Pull request by Ant3000**

### CODE REVIEW

File: GOshko/Ghw3/GOshkoHW3task2.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+b = int(input('enter bytes '))
+if b >= 1024:
+    b /= 1024
+    print("it is:", b, 'Kb')
+    if b >= 1024:
+        b /= 1024
+        b *= 1000
+        b=int(b)
```
Comment (katemalash) [inline GOshko/Ghw3/GOshkoHW3task2.py:8]: Is this operation necessary?

File: GOshko/Ghw3/GOshkoHW3task3.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+width = int(input('enter width '))
+height = int(input('enter height '))
+print('S =', width * height,'\nP =',2*(width+height))
```
Comment (katemalash) [inline GOshko/Ghw3/GOshkoHW3task3.py:3]: "*" operator has lower priority then braces, so, it should has spaces around it

File: GOshko/Ghw3/GOshkoHW3task5.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import math
+
```
Comment (katemalash) [inline GOshko/Ghw3/GOshkoHW3task5.py:2]: Why do you use this empty line?

File: GOshko/Ghw3/GOshkoHW3task6.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    if xPrint != 0:
+        if 'cats' in xPrint: print(cat)
+        if 'cars' in xPrint: print(car)
+        if 'elephants' in xPrint: print(ele)
+        if 'bicycles' in xPrint: print(bic)
+        if 'trains' in xPrint: print(tra)
+        if 'unicorns' in xPrint: print(uni)
+        xPrint = 0
+    if com == 'end':
+        break
```
Comment (katemalash) [inline GOshko/Ghw3/GOshkoHW3task6.py:80]: Perfect description!

**Review by katemalash — APPROVED** [review]
In general, Home work is done good. You can merge this pull-request after fixing some minor issues.

File: GOshko/Ghw3/GOshkoHW3task2.py (hunk 1 above)
Comment (Ant3000) [reply GOshko/Ghw3/GOshkoHW3task2.py:8]: no, it's just cosmetics for the result =)

File: GOshko/Ghw3/GOshkoHW3task5.py (hunk 3 above)
Comment (Ant3000) [reply GOshko/Ghw3/GOshkoHW3task5.py:2]: I think it's easier to see libraries, and it's not my hand, it's all PyCharm)))

**Pull request merged**